@codexteam/ui/dev/pages/components/Chart.vue

```diff
@@ -0,0 +1,450 @@
+<template>
+  <PageHeader>
+    Chart
+    <template #description>
+      A component for displaying line charts with smooth curves and interactive tooltips.
+    </template>
+  </PageHeader>
+
+  <Heading :level="3">
+    Single Line Chart
+  </Heading>
+
+  <div class="charts">
+    <div class="charts__showcase">
+      <div class="charts__controls">
```

Comment:
In your example they looks like a part of chart. It can confuse readers. Let's move them outside `__showcase`